File: src/test/java/com/janson/StudentTest.java
package com.janson;

import com.janson.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    void highestScoreTest() {
        Student student = new Student("Janson",66 , 99);
        Assertions.assertEquals(99 , student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("Janson",66 , 99);
        Assertions.assertEquals((66+99)/2 , student.getAverage());
    }
}

File: src/test/kotlin/com/janson/StudentTest.kt
package com.janson.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest(){
        val student = Students("Janson",66 , 99)
        Assertions.assertEquals(99,student.highest())
    }

    @Test
    fun averageTest() {
        val student = Students("Janson",66 , 99)
        Assertions.assertEquals((66+99)/2,student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = Students("Janson",66 , 99)
        Assertions.assertEquals('B' , student.grading())
    }
}